hack/e2e-internal/e2e-status.sh

```diff
@@ -23,7 +23,7 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 
 : ${KUBE_VERSION_ROOT:=${KUBE_ROOT}}
-: ${KUBECTL:="${KUBE_VERSION_ROOT}/cluster/kubectl.sh"}
+: ${KUBECTL:=${KUBE_VERSION_ROOT}/cluster/kubectl.sh}
```

Comment:
They aren't necessary and break github's code highlighting (see current version of file: https://github.com/mbforbes/kubernetes/blob/master/hack/e2e-internal/e2e-status.sh)